client/services/auth-service.ts

```diff
@@ -0,0 +1,48 @@
+import apiPrefix from './api-prefix';
+import { type UserToken, type ApiError } from '../../common/api-types';
+
+export async function getUser(): Promise<UserToken | null> {
+  const response = await fetch(apiPrefix + '/users');
+
+  if (!response.ok) return null;
```

Comment:
I see that you're not doing any direct error handling here. Is that just because you're operating under the assumption that if this route throws an error it will only be because a user is not signed in?